src/components/Navigation/NavigationHeadline.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { prefixClasses } from '../../utils';
+
+export const NavigationHeadline = ({ id, value }, { cssPrefix }) => {
+  const prefix = (classes, passThrough) => prefixClasses(cssPrefix, classes, passThrough);
+
+  return (
+    <h2
+      className={prefix(['text-title--caps', 'p-around--medium'])}
+      id={id}
+    >
+      {value}
+    </h2>
+  );
+};
+
+NavigationHeadline.contextTypes = { cssPrefix: React.PropTypes.string };
+
+NavigationHeadline.propTypes = {
+  /**
+   * id of the headline element
+   */
+  id: React.PropTypes.string,
+  /**
+   * headline value
+   */
+  value: React.PropTypes.string.isRequired,
```

Comment:
if there is a good reason to split this into it's own module, it would be nice to rename this prop to `title` to be consistent with the rest of the components